### PR TITLE
add decorators selectors + make it works in bitbucket.org + improve display 

### PR DESCRIPTION
**Pull request by manudss**
Add a decorators selectors (see #14 )

- [x] add the list of decorators in config.
- [x] add config to enable or disabled decorators selectors.
- [x] hide decorators selectors when no label selected or have a default label selected. 
- [x] fix validations function
- [x] make it works to bitbucket.org #21 

Fix validations function to works with customs decorators. The validation should validate, also if you have a custom decorators.  #13
Add default value for labels. To don't enable all labels by default https://github.com/smartinio/bitbucket-conventional-comments/pull/17
improve display of descriptions with tooltips :
image

Preview on bitbucket.org the decorators selectors : 
sidekick_nTpM817kCZ

### CODE REVIEW

**Comment by manudss** [comment]
@smartinio I have a first working version for decorators selectors. So we could release this first version if you are OK ? 

I could improve it later in another PR. Some idea : 

- Configure decorators for each label
- better text fields in setting for list of decorator (with tags input => need additional script). 
- ...

**Comment by manudss** [comment]
@smartinio You can now test it on bitbucket.org. I all functionality is OK for you ?

File: src/bitbucket_org.js
Comment (smartinio) [inline src/bitbucket_org.js:1]: What's this file for?

File: src/popup/popup.js [hunk 1]
```diff
@@ -15,6 +15,36 @@ const createCheckbox = async ({ label, defaultValue }) => {
...
+  if (value === undefined) {
+    state.set(id, defaultValue)
+    value = defaultValue
+  }
+  const input = document.createElement('input')
+  input.type = 'text'
+  input.id = id
+  input.value = value
+  input.addEventListener('input', getOnchangeInput(id))
+  // Créer un élément label pour le champ d'entrée de texte
```
Comment (smartinio) [inline src/popup/popup.js:35]: Why are there french comments here?

File: src/bitbucket_org.js
Comment (manudss) [reply src/bitbucket_org.js:1]: Missing files, not used finally.

File: src/popup/popup.js (hunk 1 above)
Comment (manudss) [reply src/popup/popup.js]: Thanks Removed.

**Pull request merged**